script.js

```diff
@@ -0,0 +1,17 @@
+var op1=document.getElementById('op1');
+var op2 = document.getElementById('op2');
+var op3 = document.getElementById('op3');
+var op4 = document.getElementById('op4');
+
+var app{
+   questions:[
+     { q:'question one', options: [' 1. answer one1', '2. answer two1', '3. answer three1','4. answer four1'],answer:1}
+     { q: 'question two', options: [' 1. answer one2', '2. answer two2', '3. answer three2', '4. answer four2'], answer: 1 } 
+     { q: 'question three', options: [' 1. answer one3', '2. answer two3', '3. answer three3', '4. answer four3'], answer: 1 }
+     { q: 'question four', options: [' 1. answer one4', '2. answer two4', '3. answer three4', '4. answer four4'], answer: 1 }
+    ]
+   load:function name(params) {
+     
```

Comment:
here you would write your question to the html and then write a function to handle the click by utilizing event.target.value